alyssa_mckee/function_assignments/list_to_dic.py

```diff
@@ -0,0 +1,15 @@
+def list_to_dic(list1,list2):
```

Comment:
This method is great. Since you've built out your own version of this now, then I would encourage you to look at python's built in method called .zip(). It has the same functionality and you could use that moving forward when you run into these types of situations.